docs/src/pages/demos/buttons/SpeedDials.js

```diff
@@ -0,0 +1,83 @@
+// @flow weak
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from 'material-ui/styles';
+import Button from 'material-ui/Button';
+import SpeedDialButton from 'material-ui/SpeedDial/SpeedDialButton';
```

Comment:
Sure, if that's agreed. (I'm supporting it, but lets give others a chance to have a say).

Reply:
I like the way it's exported below, so it can be imported like:

```js
import SpeedDial, { SpeedDialButton } from 'material-ui/SpeedDial';
```

This still groups imports by the component they're used with.